packages/element-web-module-api/src/models/Room.ts

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2025 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+Please see LICENSE files in the repository root for full details.
+*/
+
+import { Watchable } from "../api/watchable";
+
+/**
+ * Represents a room from element-web.
+ * @public
+ */
+export interface Room {
+    /**
+     * Id of this room.
+     */
+    id: string;
+    /**
+     * {@link Watchable} holding the name for this room.
+     */
+    name: Watchable<string>;
+    /**
+     * Get the timestamp of the last message in this room.
+     * @returns last active timestamp
+     */
+    getLastActiveTimestamp: () => number;
```

Comment:
Should this maybe also be a Watchable?

Reply:
I think this is fine for now. We could build an abstraction around the timeline later and that could just have a watchable property for this.